### ISSUE TITLE
Unicode characters lost in the wiki creator's name

### ISSUE THREAD

**Issue by MatmaRex** [MatmaRex/patchdemo#207]
Unicode characters are lost in the wiki creator's name.

This is not only embarassing, but also prevents them from deleting their wiki.

<https://meta.wikimedia.org/wiki/User:Jon_Harald_Søby>'s name was recorded as "Jon Harald Sby":
![image](https://user-images.githubusercontent.com/160413/105547030-599f6900-5cfe-11eb-9c37-d01015f75cb0.png)

**Comment by MatmaRex** [comment]
The PHP code looks correct. The file containing this data is written from `createwiki.sh`, very simply:
```
echo $CREATOR > $PATCHDEMO/wikis/$NAME/creator.txt
```
I thought this could be a problem with shell locale. To debug, I added the `locale` comment to the script, it printed:
```
LANG=
LANGUAGE=
LC_CTYPE="POSIX"
LC_NUMERIC="POSIX"
LC_TIME="POSIX"
LC_COLLATE="POSIX"
LC_MONETARY="POSIX"
LC_MESSAGES="POSIX"
LC_PAPER="POSIX"
LC_NAME="POSIX"
LC_ADDRESS="POSIX"
LC_TELEPHONE="POSIX"
LC_MEASUREMENT="POSIX"
LC_IDENTIFICATION="POSIX"
LC_ALL=
```
It should all probably be `C.UTF-8` or something like that?

**Comment by edg2s** [comment]
This would likely be fixed by #204

**Comment by edg2s** [comment]
I've manually fixed this one entry on the server for now.

**Comment by MatmaRex** [comment]
Should be fixed by #204

**Comment by jhsoby** [comment]
> Should be fixed by #204

Can confirm, everything appears correctly for the demo wiki I just created. 👍
